src/index.js

```diff
@@ -90,7 +93,12 @@ class Keyv extends EventEmitter {
 		key = this._getKeyPrefix(key);
 		const store = this.opts.store;
 		return Promise.resolve()
-			.then(() => store.delete(key));
+			.then(() => {
+				if (store instanceof Map && Array.isArray(key)) {
```

Comment:
This shouldn't only be implemented for Maps, the main benefit of using Keyv over alternatives is that it provides a consistent interface across multiple backends.

If we implement a feature only for a single backend it would break that.